keydata.go

```diff
@@ -543,10 +543,15 @@ func (d *KeyData) openWithPassphrase(passphrase string, kdf KDF) (payload []byte
 		return nil, nil, fmt.Errorf("unexpected KDF type \"%s\"", data.KDF.Type)
 	}
 	if data.Encryption != passphraseEncryption {
-		// Only AES-256-CFB is supported
+		// Only AES-CFB is supported
 		return nil, nil, fmt.Errorf("unexpected encryption algorithm \"%s\"", data.Encryption)
 	}
+	if data.KeySize > 32 {
```

Comment:
is there/should there be a minimal KeySize as well?

Reply:
This is enforced implicitly when calling aes.NewCipher, which returns an error if it's not 16, 24 or 32 bytes. The maximum is bounds checked here to avoid a potential integer overflow that can result in a panic before calling aes.NewCipher.